barman/retention_policies.py

```diff
@@ -271,6 +306,12 @@ def _backup_report(self, source):
         valid = 0
         # NOTE: reverse key orders (simulate reverse chronology)
         for bid in sorted(backups.keys(), reverse=True):
+            if backups[bid].is_incremental and len(backups) > 1:
```

Comment:
same here

Reply:
As I mentioned before, it doesn't make sense to me based on my understanding of the code.